src/org/zaproxy/zap/extension/pscanrulesBeta/CSRFCountermeasures.java

```diff
@@ -215,46 +215,46 @@ public void scanHttpResponseReceive(HttpMessage msg, int id, Source source) {
 					// Give the first FORM tag as evidence
 					evidence = formElement.getFirstElement().getStartTag().toString();
 				}
-				numberOfFormsPassed++;
-			}
-			if (sb.length() > 0) {
-				sb.append(']');
-			}
-			
-			if (!foundCsrfToken && formElements.size() > ignoredForms) {
-				//No known Anti-CSRF tokens found in a form. Not a vulnerability per-se.
-				//but alert it, as a low priority
-				String formDetails = sb.toString();
-				String tokenNamesFlattened = tokenNames.toString();
-				
-				int risk = Alert.RISK_LOW;
-				String desc = Constant.messages.getString("pscanbeta.noanticsrftokens.desc");
-				String extraInfo = Constant.messages.getString("pscanbeta.noanticsrftokens.alert.extrainfo", tokenNamesFlattened, formDetails);
-				if (hasSecurityAnnotation) {
-					risk = Alert.RISK_INFO;
-					extraInfo = Constant.messages.getString("pscanbeta.noanticsrftokens.extrainfo.annotation");
+
+				if (sb.length() > 0) {
+					sb.append(']');
 				}
 				
-			    Alert alert = new Alert(getPluginId(), risk, Alert.CONFIDENCE_MEDIUM,  getName());
-			    alert.setDetail(
-			    			desc + "\n"+getDescription(), 
-				    		msg.getRequestHeader().getURI().toString(),
-				    		"",  //parameter: none.
-				    		"", 
-				    		extraInfo,
-				    		getSolution(), 
-				            getReference(), 
-							evidence,	// Evidence
-							352, // CWE-352: Cross-Site Request Forgery (CSRF)
-							9,	// WASC Id
-				            msg);
-
-			    parent.raiseAlert(id, alert);
+				if (!foundCsrfToken && formElements.size() > ignoredForms) {
+					//No known Anti-CSRF tokens found in a form. Not a vulnerability per-se.
+					//but alert it, as a low priority
+					String formDetails = sb.toString();
+					String tokenNamesFlattened = tokenNames.toString();
+					
+					int risk = Alert.RISK_LOW;
+					String desc = Constant.messages.getString("pscanbeta.noanticsrftokens.desc");
+					String extraInfo = Constant.messages.getString("pscanbeta.noanticsrftokens.alert.extrainfo", tokenNamesFlattened, formDetails);
+					if (hasSecurityAnnotation) {
+						risk = Alert.RISK_INFO;
+						extraInfo = Constant.messages.getString("pscanbeta.noanticsrftokens.extrainfo.annotation");
+					}
+					
+				    Alert alert = new Alert(getPluginId(), risk, Alert.CONFIDENCE_MEDIUM,  getName());
+				    alert.setDetail(
+				    			desc + "\n"+getDescription(), 
+					    		msg.getRequestHeader().getURI().toString(),
+					    		"",  //parameter: none.
+					    		"", 
+					    		extraInfo,
+					    		getSolution(), 
+					            getReference(), 
+								evidence,	// Evidence
+								352, // CWE-352: Cross-Site Request Forgery (CSRF)
+								9,	// WASC Id
+					            msg);
+	
+				    parent.raiseAlert(id, alert);
+				}
 			}
 		}
-		if (logger.isDebugEnabled()) {
-			logger.debug("\tScan of record " + id + " took " + (System.currentTimeMillis() - start) + " ms");
-		}
+			if (logger.isDebugEnabled()) {
```

Comment:
The previous indentation is/was fine (it's the method closing brace, line 258, that's not properly indented).